README.md

```diff
@@ -1,2 +1,44 @@
-# church-conference-capstone
-This is a project for an annual church conference. 
+# Project Name
+
+Faith Apostolic Annual Conference Website.
+
+![screenshot](./app_screenshot.png)
```

Comment:
- [ ] Image link is broken. Please update the image path with the correct path for the screenshot image.